core/templates/dev/head/player/PlayerServices.js

```diff
@@ -535,69 +542,76 @@ oppia.factory('oppiaPlayerService', [
           answer, currentInteraction.id, currentInteraction.customization_args);
       }
     },
-    submitAnswer: function(answer, successCallback, delayParamUpdates) {
+    getClassificationResult: function(answer) {
```

Comment:
For convention, it probably should be named after an action. The promise being returned is so that the caller can check-up on the process (so it should definitely be mentioned in a docstring). Perhaps ``classifyAnswer()``?